<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEventWindow</class>
 <widget class="QDialog" name="AddEventWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>371</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>439</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>439</width>
    <height>371</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Uniform Rnd Medium</family>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Add Event</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color:rgb(255, 255, 255);
}

QMessageBox {
    background-color: rgb(255, 255, 255);
	font: 57 10pt &quot;Uniform Rnd Medium&quot;;
}

/*Calendars*/
QCalendarWidget{
	background-color: rgb(255, 255, 255);
	width: 30px;
	height: 50px;
}

QCalendarWidget QToolButton {
  	color: rgb(233, 66, 146);
	font: 8pt &quot;Comfortaa&quot;;
  	background-color: rgb(255, 255, 255);
  }

  QCalendarWidget QMenu {
	color: rgb(233, 66, 146);
	background-color: rgb(255, 255, 255);
	font: 14pt &quot;Comfortaa&quot;;
}

QCalendarWidget QSpinBox { 
	font: 57 14pt &quot;Uniform Rnd Medium&quot;;
  	color: rgb(233, 66, 146); 
  	background-color: rgb(255, 255, 255); 
  	selection-background-color: rgb(136, 136, 136);
  	selection-color: rgb(255, 255, 255);
  }

QCalendarWidget QSpinBox ::up-button{ 
    subcontrol-origin: border;
    subcontrol-position: top right; 
	background-color: white;
	border: 0px;
}

QCalendarWidget QSpinBox ::up-arrow{ 
	image: url(:/ui/down.png);
}

  /* header row */
  QCalendarWidget QWidget 
	{ 
	font: 57 12pt &quot;Uniform Rnd Medium&quot;;
	alternate-background-color: rgb(255, 255, 255);
	background-color: rgb(255, 255, 255);
	}
   
  /* normal days */
  QCalendarWidget QAbstractItemView:enabled {
	font: 57 8pt &quot;Uniform Rnd Medium&quot;;  
  	color: rgb(63, 63, 63);  
  	background-color: rgb(255, 255, 255);  
  	selection-background-color: rgb(84, 163, 219); 
  	selection-color: rgb(255, 255, 255); 
  }

QCalendarWidget QAbstractItemView:disabled  { 
color: rgb(247, 247, 247); 
}</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color:rgb(233, 66, 146);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(255, 255, 255);
	color: rgb(255, 255, 255);
	font: 57 10pt &quot;Uniform Rnd Medium&quot;;
</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color: rgb(255, 255, 255);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: rgb(255, 255, 255);
	color: rgb(68, 68, 68);
	font: 57 10pt &quot;Uniform Rnd Medium&quot;;
</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>371</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="Priority">
        <property name="styleSheet">
         <string notr="true">font: 57 10pt &quot;Uniform Rnd Medium&quot;;</string>
        </property>
        <property name="text">
         <string>Priority</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="PriorityBox">
        <property name="styleSheet">
         <string notr="true">QComboBox{
font: 57 8pt &quot;Uniform Rnd Medium&quot;;
color: rgb(233, 66, 146);
background-color:rgb(245, 245, 245);
border: 0px;
}

QComboBox::drop-down {
    image: url(:ui/down.png);
	color: rgb(233, 66, 146);
    width:30px;
    height:15px;
    subcontrol-position: right middle;
    background-color:  rgb(245, 245, 245);
    border-style: outrange;
    border-width: 0px;
    border-color: white;
}</string>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtBottom</enum>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Low</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="Name">
        <property name="styleSheet">
         <string notr="true">font: 57 10pt &quot;Uniform Rnd Medium&quot;;</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NameEdit">
        <property name="styleSheet">
         <string notr="true">color: rgb(233, 66, 146);
background-color:rgb(245, 245, 245);
border: 0px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="Place">
        <property name="styleSheet">
         <string notr="true">font: 57 10pt &quot;Uniform Rnd Medium&quot;;</string>
        </property>
        <property name="text">
         <string>Place</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PlaceEdit">
        <property name="styleSheet">
         <string notr="true">color: rgb(233, 66, 146);
background-color:rgb(245, 245, 245);
border: 0px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="Accomanion">
        <property name="styleSheet">
         <string notr="true">font: 57 10pt &quot;Uniform Rnd Medium&quot;;</string>
        </property>
        <property name="text">
         <string>Accompanion</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AccEdit">
        <property name="styleSheet">
         <string notr="true">color: rgb(233, 66, 146);
background-color:rgb(245, 245, 245);
border: 0px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="StartDate">
        <property name="styleSheet">
         <string notr="true">font: 57 10pt &quot;Uniform Rnd Medium&quot;;</string>
        </property>
        <property name="text">
         <string>Start Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="StartDateTimeEdit">
        <property name="styleSheet">
         <string notr="true">QDateTimeEdit
{
	font: 57 8pt &quot;Uniform Rnd Medium&quot;;
	color: rgb(233, 66, 146);
	background-color:rgb(245, 245, 245);
	border: 0px;
}

QDateTimeEdit::drop-down {
    image: url(:/Rsc/down.png);
	color: rgb(233, 66, 146);
    width:30px;
    height:15px;
    subcontrol-position: right middle;
    background-color:  rgb(245, 245, 245);
    border-style: outrange;
    border-width: 0px;
    border-color: white;
}</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="time">
         <time>
          <hour>18</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="EndDate">
        <property name="styleSheet">
         <string notr="true">font: 57 10pt &quot;Uniform Rnd Medium&quot;;</string>
        </property>
        <property name="text">
         <string>End Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="EndDateTimeEdit">
        <property name="styleSheet">
         <string notr="true">QDateTimeEdit
{
	font: 57 8pt &quot;Uniform Rnd Medium&quot;;
	color: rgb(233, 66, 146);
	background-color:rgb(245, 245, 245);
	border: 0px;
}

QDateTimeEdit::drop-down {
    image: url(:/Rsc/down.png);
	color: rgb(233, 66, 146);
    width:30px;
    height:15px;
    subcontrol-position: right middle;
    background-color:  rgb(245, 245, 245);
    border-style: outrange;
    border-width: 0px;
    border-color: white;
}</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QCheckBox" name="ReminderButton">
        <property name="styleSheet">
         <string notr="true">QCheckBox {
font: 57 10pt &quot;Uniform Rnd Medium&quot;;
}

QCheckBox::indicator {
	color:rgb(233, 66, 146);
}
</string>
        </property>
        <property name="text">
         <string>Reminder</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="ReminderDateTimeEdit">
        <property name="styleSheet">
         <string notr="true">QDateTimeEdit
{
	font: 57 8pt &quot;Uniform Rnd Medium&quot;;
	color: rgb(233, 66, 146);
	background-color:rgb(245, 245, 245);
	border: 0px;
}

QDateTimeEdit::disabled{
	font: 57 8pt &quot;Uniform Rnd Medium&quot;;
	color: rgb(154, 154, 154);
	background-color:rgb(245, 245, 245);
	border: 0px;
}

QDateTimeEdit::drop-down {
    image: url(:/Rsc/down.png);
	color: rgb(233, 66, 146);
    width:30px;
    height:15px;
    subcontrol-position: right middle;
    background-color:  rgb(245, 245, 245);
    border-style: outrange;
    border-width: 0px;
    border-color: white;
}

QDateTimeEdit::drop-down::disabled {
    image: url(:/Rsc/down-disabled.png);
	color: rgb(233, 66, 146);
    width:30px;
    height:15px;
    subcontrol-position: right middle;
    background-color:  rgb(245, 245, 245);
    border-style: outrange;
    border-width: 0px;
    border-color: white;
}</string>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>6</day>
         </datetime>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
